### DISCUSSION TITLE
vAmigaWeb ‘Paula Volume Slider’

### DISCUSSION THREAD

**Discussion by aZtOcKdOg** [vAmigaWeb/vAmigaWeb#161] [Ideas]
(Oops I posted this over at vAmiga repo accidentally but he did provide some good info!)

ORIGINAL MSG:

Hi, I have been doing a bit of streaming across Amiga emulation distros and the 'Paula Volume Control' of WinUAE / Amiberry has come in extremely handy for levelling audio levels between digital inputs on the stream box!

This volume would be in the gain stage BEFORE the iOS / iPadOS audio volume...

I guess if that hasn't been implemented by the vAmiga core than it would be difficult to jack into but if it is there somewhere then perhaps just a gui slider or numerical indicator dialog.

Just quick thoughts and great updates btw! Lets chat again

**Comment by mithrendal** [comment]
Ups I somehow missed this one … just read about it today … yes makes sense … should be not too difficult to implement…
